### PR TITLE
[DataViz] I18n for DataViz

### PR DESCRIPTION
**Pull request by ajpal**
# Description
No user-visible changes, just moves strings to i18n files instead of being hardcoded.

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by Hamms — APPROVED** [review]
this looks great!

**Review by jmkulwik — APPROVED** [review]

File: apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx [hunk 1]
```diff
@@ -153,13 +158,20 @@ class VisualizerModal extends React.Component {
...
-              onChange={event => this.setState({chartType: event.target.value})}
+              onChange={event =>
+                this.setState({chartType: parseFloat(event.target.value)})
+              }
             />
 
-            {this.state.chartType === 'Histogram' && (
+            {this.state.chartType === ChartType.HISTOGRAM && (
               <div style={styles.input}>
                 <label style={rowStyle.description}>Bucket Size</label>
```
Comment (epeach) [inline apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx:176]: Does this one need to i18n too?

**Review by epeach — APPROVED** [review]
Just one place that might need to be i18n too. Other than that, looks good!

File: apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx (hunk 1 above)
Comment (ajpal) [reply apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx:176]: 🤦‍♀ Yes, my bad. Will do in a quick follow up.

**Pull request merged**